js/modules/Debug/Debug.js

```diff
@@ -386,7 +386,7 @@ window.GLPI.Debug = new class Debug {
     refreshWidgetButtons() {
         // Server performance
         const server_perf = this.initial_request.server_performance;
-        const memory_usage_mio = (server_perf.memory_usage / 1024 / 1024).toFixed(2);
+        const memory_usage_mio = +(server_perf.memory_usage / 1024 / 1024).toFixed(2);
```

Comment:
Maybe we should replace `/ 1024 / 1024` by `/ 1000 / 1000` if we use the `MB` unit.

Reply:
I put back MiB values, because this is the unit used by PHP for the memory limit configuration.

With a `memory_limit=128M` value, if we expose MB, the max value will seems weird.
![image](https://github.com/glpi-project/glpi/assets/33253653/424029da-51cd-4162-8f25-e3220e53bb26)

If we expose MiB, it looks more correct:
![image](https://github.com/glpi-project/glpi/assets/33253653/c91c4f7a-a4c6-4685-b3ff-54c27da59d96)